phpBB/includes/acp/acp_database.php

```diff
@@ -95,29 +95,41 @@ function main($id, $mode)
 							case 'mysqli':
 							case 'mysql4':
 							case 'mysql':
-								$extractor = new mysql_extractor($format, $filename, $time, $download, $store);
+								$extractor = new \phpbb\db\extractor\mysql_extractor(
```

Comment:
please define a dbal.extractor service with a dbal.extractor.factory like we did for the dbal tools: https://github.com/phpbb/phpbb/blob/master/phpBB/config/default/container/services_db.yml#L12-L21